networkx/linalg/attrmatrix.py

```diff
@@ -327,25 +328,25 @@ def attr_sparse_matrix(
     G : graph
         The NetworkX graph used to construct the NumPy matrix.
 
-    edge_attr : str, optional
+    edge_attr : str, optional (default: number of edges for each matrix element)
         Each element of the matrix represents a running total of the
         specified edge attribute for edges whose node attributes correspond
         to the rows/cols of the matrix. The attribute must be present for
         all edges in the graph. If no attribute is specified, then we
         just count the number of edges whose node attributes correspond
         to the matrix element.
 
-    node_attr : str, optional
+    node_attr : str, optional (default: use nodes of the graph)
         Each row and column in the matrix represents a particular value
         of the node attribute.  The attribute must be present for all nodes
         in the graph. Note, the values of this attribute should be reliably
         hashable. So, float values are not recommended. If no attribute is
         specified, then the rows and columns will be the nodes of the graph.
 
-    normalized : bool, optional
+    normalized : bool, optional (default: False)
         If True, then each row is normalized by the summation of its values.
 
-    rc_order : list, optional
+    rc_order : list, optional (default: order of nodes in G)
         A list of the node attribute values. This list specifies the ordering
         of rows and columns of the array. If no ordering is provided, then
         the ordering will be random (and also, a return value).
```

Comment:
While we're making this docstring more clear, can we change this use of "random"? It's arbitrary, not random, and defaults to the order of iterating over nodes of the graph.

Reply:
Nice -- I reworded this to make it more clear and remove the term "random" when it is very much not random. :)